velox/exec/Task.cpp

```diff
@@ -1002,7 +1002,7 @@ bool Task::isFinishedLocked() const {
   return (state_ == TaskState::kFinished);
 }
 
-void Task::updateBroadcastOutputBuffers(int numBuffers, bool noMoreBuffers) {
+bool Task::updateBroadcastOutputBuffers(int numBuffers, bool noMoreBuffers) {
```

Comment:
I don't think this change is needed (or correct) and the return value doesn't appear to be used.

Reply:
I am intending to use here @mbasmanova . If return value is false and task was aborted or cancelled, then we return from createOrUpdateTask with TaskInfo as return value.
https://github.com/prestodb/presto/blob/master/presto-native-execution/presto_cpp/main/TaskManager.cpp#L339

Reply:
I see. Thank you for clarifying.